packages/charting/src/components/Legends/Legends.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react';
+
+import { IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { HoverCard, IExpandingCardProps } from 'office-ui-fabric-react/lib/HoverCard';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { ResizeGroup } from 'office-ui-fabric-react/lib/ResizeGroup';
+import { DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
```

Comment:
This class should accept a `theme` prop rather than using the default theme. Please see other charting components (e.g. LineChart) for examples.